config/development.json

```diff
@@ -1,15 +1,15 @@
 {
   "basepath": "/",
-  "url": "https://hoverboard-dev.firebaseapp.com/",
-  "analytics": "UA-43643469-8",
+  "url": "devfestmaputo.firebaseapp.com",
+  "analytics": "UA-43643469-9",
   "prefix": "/",
   "firebase": {
-    "apiKey": "AIzaSyBu9LI-hY4NfUOPJJKuV3BfQm7edLr2-aE",
-    "authDomain": "hoverboard-dev.firebaseapp.com",
-    "databaseURL": "https://hoverboard-dev.firebaseio.com",
-    "projectId": "hoverboard-dev",
-    "storageBucket": "hoverboard-dev.appspot.com",
-    "messagingSenderId": "1083195713896"
+    "apiKey": "AIzaSyBP7dIaa9ELiZujHIeL0d-W9JwpLmkN3xw",
+    "authDomain": "hoverboard-master.firebaseapp.com",
+    "databaseURL": "https://devfestmaputo.firebaseio.com",
+    "projectId": "devfestmaputor",
```

Comment:
I think our projectId is `devfestmaputo`

Reply:
yes, it is